label_studio/tests/webhooks/test_webhooks.py

```diff
@@ -352,7 +352,7 @@ def test_webhooks_for_action_delete_tasks_annotations(configured_project, busine
 
 # ACTION: DELETE TASKS
 @pytest.mark.django_db
-def test_webhooks_for_action_delete_tasks_annotations(configured_project, business_client, organization_webhook):
+def test_webhooks_for_action_delete_tasks(configured_project, business_client, organization_webhook):
```

Comment:
drive-by: I noticed there's another test with the same name as the old name (probably copypasta) so this wasn't running.